extensions/amp-render/1.0/amp-render.js

```diff
@@ -212,7 +214,51 @@ export class AmpRender extends BaseElement {
       },
       'onReady': () => {
         this.toggleLoading(false);
-        this.togglePlaceholder(false);
+        if (this.element.getAttribute('layout') !== Layout.CONTAINER) {
+          this.togglePlaceholder(false);
+          return;
+        }
```

Comment:
I believe we want to toggle the placeholder off this early even in the resize case, as rendered content will be available (just potentially clipped):
```suggestion
        this.togglePlaceholder(false);
        if (this.element.getAttribute('layout') !== Layout.CONTAINER) {
          return;
        }
```